cmd/enry/main.go

```diff
@@ -200,7 +204,8 @@ func printPercents(fSummary map[string][]string, buff *bytes.Buffer, mode string
 		fileValues      = make(map[string]float64)
 	)
 	for fType, files := range fSummary {
-		val, err := reducer(files)
+		//FIXME(bzz): all files here have relative paths
```

Comment:
```suggestion
		// FIXME(bzz): all files here have relative paths
```

Reply:
More importantly, what is intended to be fixed? Is the issue that we should do something different if not all the files are relative? Or that we should add more tests covering absolute paths? Or something else?

Reply:
that was `FIXME` in it's true spirit - not mean to be committed and was already fixed